src/common/src/system_param/mod.rs

```diff
@@ -40,19 +42,24 @@ macro_rules! for_all_undeprecated_params {
             { data_directory, String, "hummock_001".to_string() },
             { backup_storage_url, String, "memory".to_string() },
             { backup_storage_directory, String, "backup".to_string() },
+            $({ $field, $type, $default },)*
         }
     };
 }
 
-// Only includes deprecated params. Used to define key constants.
+// Includes all params.
 // Macro input is { field identifier, type, default value }
-macro_rules! for_all_deprecated_params {
+macro_rules! for_all_params {
     ($macro:ident) => {
-        $macro! {}
+        for_all_undeprecated_params!(
+            $macro /* Define future deprecated params here, such as
+                    * ,{ backup_storage_directory, String, "backup".to_string() } */
```

Comment:
This really works but seems a little bit strange to pass deprecated params to the macro of `undeprecated_params`. 🤣

Reply:
I really can't think of another way to put two `for_all_xxx` mcros together 😢